### PR TITLE
swagger setup chore

### PR DESCRIPTION
**Pull request by bolubee101**
#### Packages installed
- @nestjs/swagger 
- swagger-ui-express

#### Setup demo
image

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
   const app = await NestFactory.create(AppModule);
+
+  const swaggerConfig = new DocumentBuilder()
+    .setTitle('Talking Stage app API')
+    .setDescription('Talking Stage app API')
+    .setVersion('1.0')
+    .build();
+
+  const doc = SwaggerModule.createDocument(app, swaggerConfig);
+  SwaggerModule.setup('api', app, doc);
```
Comment (Emmanuerl) [inline src/main.ts]: rename `api` to `api-spec`

**Review by Emmanuerl — CHANGES_REQUESTED** [review]
Also, I hope this swagger documentation takes into cognisance the DTOs and DB entities that will be created in this project?

File: src/main.ts (hunk 1 above)
Comment (bolubee101) [reply src/main.ts]: Renamed.

**Review by Emmanuerl — APPROVED** [review]

**Comment by bolubee101** [comment]
> Also, I hope this swagger documentation takes into cognisance the DTOs and DB entities that will be created in this project?

It does. Whenever DTO's are created, they can be documented using the  `ApiProperty`  decorator

**Comment by Emmanuerl** [comment]
> > Also, I hope this swagger documentation takes into cognisance the DTOs and DB entities that will be created in this project?
> 
> It does. Whenever DTO's are created, they can be documented using the `ApiProperty` decorator

Ideally, you don't need to add the ApiProperty decorator, it'll actually pick it up, if i notice you didn't add that i'll just add it myself, what you've done is already okay

**Pull request merged**